test/shared/npmCommand.test.ts

```diff
@@ -188,11 +199,13 @@ describe('should find the node executable', () => {
   });
 
   it('finds node binary inside sfdx bin folder and runs npm show command', () => {
-    const npmMetadata = new NpmModule(MODULE_NAME, undefined, __dirname).show(DEFAULT_REGISTRY);
+    const npmMetadata = new NpmModule(MODULE_NAME, undefined, dirname(fileURLToPath(import.meta.url))).show(
+      DEFAULT_REGISTRY
+    );
     expect(accessSyncStub).to.have.been.calledOnce;
     expect(existsSyncStub).to.have.been.calledTwice;
     expect(osTypeStub).to.have.been.calledOnce;
-    expect(realpathSyncStub).to.have.been.calledOnce;
+    // expect(realpathSyncStub).to.have.been.calledOnce;
```

Comment:
Were these a TODO, do the tests not work, has the logic changed, or do these need to be removed?